examples/k8s-upgrade/v1.7.9-win.json.env

```diff
@@ -1,2 +1,6 @@
 ACSE_POSTDEPLOY=examples/k8s-upgrade/k8s-upgrade.sh
+<<<<<<< HEAD:examples/k8s-upgrade/v1.6.json.env
```

Comment:
this file has to be resolved

Reply:
which one should I keep? 1.7.10 or 1.8.2?

Reply:
should be 1.8.2